lib/process/watch.go

```diff
@@ -214,10 +211,9 @@ func (p *Process) reloadAuthGatewayConfig(authGatewayConfig storage.AuthGateway)
 	return nil
 }
 
-// startWatchingReloadEvents launches watcher that listens for reload events
-// and restarts the process.
-func (p *Process) startWatchingReloadEvents(ctx context.Context, client *kubernetes.Clientset) error {
-	go func() {
+// runReloadEventsWatch watches reload events and restarts the process.
+func (p *Process) runReloadEventsWatch(client *kubernetes.Clientset) clusterService {
```

Comment:
Same as above two comments about "cluster services". IIRC I introduced this "cluster service" thing specifically for things that should run only in active gravity site. This should run in all.

Reply:
What's the problem with having more `cluster services`? I understand there're services that are only supposed to run on the leader which they continue to do - I just generalized the rest of similar "services" to have the same signature.